modules/snooker/src/main/ts/ephox/snooker/api/TableOperations.ts

```diff
@@ -363,3 +363,4 @@ export const pasteColsAfter = run(opPasteColsAfter, onPasteByEditor, Fun.noop, F
 export const pasteRowsBefore = run(opPasteRowsBefore, onPasteByEditor, Fun.noop, Fun.noop, Generators.modification);
 export const pasteRowsAfter = run(opPasteRowsAfter, onPasteByEditor, Fun.noop, Fun.noop, Generators.modification);
 export const getColumnType = opGetColumnType;
+export const getUniqueColumns = uniqueColumns;
```

Comment:
Do we really want this exposed on the public snooker API? It might be better to move this function to an internal helper/utils module and then reuse it that way.